tests/conf/sen12ms_s1.yaml

```diff
@@ -2,11 +2,11 @@ experiment:
   task: "sen12ms"
   module:
     loss: "focal"
-    segmentation_model: "fcn"
+    model: "fcn"
     num_filters: 1
-    encoder_name: "resnet18"
-    encoder_weights: null
-    encoder_output_stride: 16
+    backbone: "resnet18"
+    weights: null
+    backbone_output_stride: 16
```

Comment:
Wait a minute, is the `backbone_output_stride` parameter actually used? If not, let's just remove it from all conf files.

Reply:
I think it is not used, "encoder_output_stride" is a valid argument for the pytorch-segmentation-models model input, but we do not mention it elsewhere explicitly.

Reply:
Gotcha. If we don't use it, I would just remove it.